extensions/federation/README.md

```diff
@@ -92,7 +92,28 @@ schema:
 
 ## Lists of resources
 
-Clients will assume that all lists of resources (e.g. collections, processes, jobs, ...) are the combined from all back-ends listed in `GET /`. Federated APIs can expose if any of the back-ends is not available and thus is not part of the response.
+Clients will assume that all lists of resources are the a combination of all back-ends listed in `GET /`.
+Federated APIs can expose if any of the back-ends is not available and thus is not part of the response.
+
+Applies to:
+
+- `GET /collections`
+- `GET /processes`
+- `GET /file_formats`
+- `GET /process_graphs`
+- `GET /jobs`
+- `GET /jobs/{job_id}`
+- `GET /jobs/{job_id}/logs`
+- `GET /jobs/{job_id}/results`
+- `GET /services`
+- `GET /services/{service_id}/logs`
+
+The following two endpoints are not extensible and as such the additional field can't be added:
+
+- `GET /udf_runtimes`
+- `GET /service_types`
+
+*This is due to the fact that these two endpoints return an object with a key-value-pair per UDF runtime / service type and adding a key-value-pair for federation purposes would be interpreted as a new UDF runtime or service type by clients.*
```

Comment:
> The following two endpoints ...
> ...
> *This is due to the fact ...*


It took me couple of attempts to understand what you explain here.

proposal to simplify:

> The following endpoints define the resources (UDF runtimes / service types) at the top level of their response as key-value pairs. Consequently they are not extensible with additional properties for federation purposes.
> - `GET /udf_runtimes`
> - `GET /service_types`

Reply:
Thanks, updated.